ReportPortal.SpecFlowPlugin/ReportPortalHooks.cs

```diff
@@ -167,19 +197,19 @@ public void BeforeScenario()
                     Name = this.ScenarioContext.ScenarioInfo.Title,
                     Description = this.ScenarioContext.ScenarioInfo.Description,
                     StartTime = DateTime.UtcNow,
-                    Type = TestItemType.Step,
```

Comment:
Please don't change item type, user will loose the entire history for tests. It seems we are waiting when RP will support BDD style reporting.

Reply:
Reverted.